precompiled/serde_derive/Cargo.toml

```
@@ -26,6 +27,11 @@ proc-macro2 = "1"
 quote = "1"
 syn = "2.0.28"
 
+[dependencies]
```

```suggestion
[target.'cfg(any(feature = "from_source", not(all(target_arch = "x86_64", target_os = "linux", target_env = "gnu"))))'.dependencies]
```

This needs to be gated otherwise it pulls the dependencies slowing down ?

---

oops scrap that - these need to be optional and then under [features] triggered to be included

---

```suggestion
[dependencies]
proc-macro2 = { version = "1", optional = true }
quote = { version = "1", optional = true }
syn = "2.0.28", optional = true }

[features]
from_source = ["proc-macro2", "quote", "syn"]
```